bridged-nft-token/src/metadata.rs

```diff
@@ -0,0 +1,41 @@
+use crate::*;
+
+#[derive(BorshDeserialize, BorshSerialize, Serialize, Deserialize, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub struct NFTMetadata {
+    pub spec: String,              // required, essentially a version like "nft-1.0.0"
+    pub name: String,              // required, ex. "Mosaics"
+    pub symbol: String,            // required, ex. "MOSIAC"
```

Comment:
```suggestion
    pub symbol: String,            // required, ex. "MOSAIC"
```

Reply:
I removed this struct and used the struct from the standard lib